.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
-elm_stuff
+elm-stuff
```

Comment:
Just checking before I merge, your `elm-stuff` folder isn't called `elm_stuff` is it @XertroV ?